CHANGES/9200.bugfix

```diff
@@ -0,0 +1 @@
+Moved DB fields encryption from pulp_api to pulp_database_config role
```

Comment:
It would be preferable to tell the users what bug this actually fixes. Under what circumstances did what task break. And to tell them that it is now generated for all of the pulp services.

Reply:
Wait, that isn't what role it is in. It is in pulp_database_config. That is wrong, users may have pulp_database_config and pulp-api on 2 separate hosts.